sciencer/filters/filter_by_field_of_study.py

```diff
@@ -0,0 +1,20 @@
+"""Filters paper by words in abstract
+"""
+from .filter import Filter
+from ..models import Paper
+
+
+class FilterByFieldOfStudy(Filter):
+    """Class that encapsulates the filter of paper by its fields of study"""
+
+    def __init__(self, field_of_study: str) -> None:
+        super().__init__(policies=[])
+        self.__field_of_study: str = field_of_study
+
+    def is_valid(self, paper: Paper) -> bool:
+        if paper.fields_of_study is None:
+            return False
```

Comment:
Shouldn't we be more permissive? If no field of study is defined, then we do not know if it is or not from the desired field.

Reply:
I agree that the developer should be able to control the default validity when the necessary properties are not available. 
However, I would make the default more restrictive to reduce the search time. 

A new issue for this feature was created #24